example/gluon/estimator_example/alexnet.py

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This example is inspired from
+# https://github.com/d2l-ai/d2l-en/blob/master/chapter_convolutional-neural-networks/alexnet.md
+# Model definition is from
+# https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/alexnet.py
+
+
+import os
+import sys
+import argparse
+import mxnet as mx
+from mxnet import gluon
+from mxnet.gluon import nn, data
+from mxnet.gluon.block import HybridBlock
+from mxnet.gluon.estimator import estimator, event_handler
+
+def parse_args():
+    '''
+    Command Line Interface
+    '''
+    parser = argparse.ArgumentParser(description='Train ResNet18 on Fashion-MNIST')
+    parser.add_argument('--batch-size', type=int, default=128,
+                        help='training batch size per device (CPU/GPU).')
+    parser.add_argument('--num-epochs', type=int, default=1,
+                        help='number of training epochs.')
+    parser.add_argument('--input-size', type=int, default=224,
+                        help='size of the input image size. default is 224')
+    parser.add_argument('--lr', type=float, default=0.001,
+                        help='learning rate. default is 0.001')
+    parser.add_argument('-j', '--num-workers', default=None, type=int,
+                        help='number of preprocessing workers')
+    opt = parser.parse_args()
+    return opt
+
+class AlexNet(HybridBlock):
```

Comment:
you can fetch this from gluon-cv

Reply:
Yes. Since the gluon model zoo already has alexnet and resnet, I will import it from there.

Reply:
@abhinavs95  You can also use default arguments  eg. learning rate , num_workers to make the example simple for the beginners

Reply:
will do, thanks!